### ISSUE TITLE
Problems with k8s-platform masters in sandbox & staging

### ISSUE THREAD

**Issue by stephen-soltesz** [m-lab/k8s-support#85]
Cause is unknown.

Related open issues that should be addressed after resolving this:

* https://github.com/m-lab/dev-tracker/issues/142

Observed symptoms:

 * in sandbox, nodes running setup_k8s.sh fail to request a k8s token due to 'Bad Gateway' -- this is usualy the result of the epoxy server being unable to contact the token server on the k8s-platform mater. This may be due to the token server not running, not being accessible, or the two networks being otherwise disconnected.

* in staging, nodes running setup_k8s.sh succeed in getting a token, and apparently join the cluster (success message in /tmp/k8s_setup.log). However, when visiting the mlab-staging master node and running `kubectl get nodes` only the master nodes are reported. It appears that the master does not know about the node that just joined. This does not make sense.

As a consequence, while the machines boot and appear to join the cluster they cannot be managed by the k8s master, so no daemonsets or containers load.

**Comment by evfirerob** [comment]
Staging: looks like by running a `systemctl restart kubelet` some time after `kubeadm join` the node correctly joins the cluster and it appears in the output of `kubectl get nodes`.

In particular, there is an error in kubelet's startup log that is not there if kubelet is started later:
```
Dec 05 12:45:21 mlab4.lga05.measurement-lab.org kubelet[1776]: W1205 12:45:21.466142    1776 cni.go:188] Unable to update cni config: No networks found in /etc/cni/net.d
Dec 05 12:45:21 mlab4.lga05.measurement-lab.org kubelet[1776]: W1205 12:45:21.467789    1776 cni.go:188] Unable to update cni config: No networks found in /etc/cni/net.d

```

I suspect the change at https://github.com/m-lab/k8s-support/commit/de3fc481016009150dee0185203e2f0d29b86db4 might fix the issue.

**Comment by pboothe** [comment]
Staging we think is fixed (PRs #87 #86)

**Comment by stephen-soltesz** [comment]
Done.

Additionally https://github.com/m-lab/k8s-support/pull/88
and https://github.com/m-lab/k8s-support/pull/89 which migrates the k8s platform region to us-east1 to match the default region for AppEngine in mlab-sandbox for access to the internal load balancer from the epoxy server.
